datastore/multi.go

```diff
@@ -23,11 +23,13 @@ var (
 	headStoreKey   = rootStoreKey.ChildString("heads")
 	blockStoreKey  = rootStoreKey.ChildString("blocks")
 	peerStoreKey   = rootStoreKey.ChildString("ps")
+	encStoreKey    = rootStoreKey.ChildString("enc")
 )
 
 type multistore struct {
 	root   DSReaderWriter
 	data   DSReaderWriter
+	enc    DSReaderWriter
```

Comment:
it sounds better, but less relevant and 3 bytes longer.
I not happy myself with "enc" as it might not be immediatly clear, but tend to like it slightly more than "crypto". If other also see "crypto" being a better much (or anything else), no problem.